api/Makefile

```diff
@@ -39,8 +39,10 @@ gen: clean            ## Generate files.
 			--additional-initialism=pt \
 			--additional-initialism=qan \
 			--additional-initialism=rds \
-			--additional-initialism=sql ; \
+			--additional-initialism=psmdb \
+			--additional-initialism=sql & ); \
 	done
+	wait
```

Comment:
#sequentially:
make -C api gen  48.70s user 13.75s system 225% cpu 27.731 total
make -C api gen  50.94s user 14.08s system 214% cpu 30.354 total
make -C api gen  48.89s user 13.23s system 226% cpu 27.392 total
#in parallel
make -C api gen  90.04s user 14.93s system 198% cpu 52.924 total
make -C api gen  91.50s user 15.32s system 198% cpu 53.899 total
make -C api gen  90.09s user 14.93s system 197% cpu 53.251 total